examples/s2_field_combination_extern_rf_train/re_add_raster_dataset_from_upload.ipynb

```diff
@@ -0,0 +1,5550 @@
+{
```

Comment:
Maybe you should add a small markdown file to this example folder that briefly describes the contents?